features/mbedtls/targets/TARGET_STM/sha256_alt.h

```diff
@@ -0,0 +1,127 @@
+/*
+ *  sha256_alt.h SHA-256 hash
+ *******************************************************************************
+ *  Copyright (C) 2017, STMicroelectronics
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+#ifndef MBEDTLS_SHA256_ALT_H
+#define MBEDTLS_SHA256_ALT_H
+
+#if defined (MBEDTLS_SHA256_ALT)
+#include "mbedtls/platform.h"
+#include "mbedtls/config.h"
```

Comment:
Is this include needed? Please check what header file includes are needed in `mbedtls_alt.h` file and what can be moved to `mbedtls_alt.c` file